### PR TITLE
Add cargo-fuzz related files.

### PR DESCRIPTION
**Pull request by frewsxcv**
https://github.com/yurydelendik/wasmparser.rs/issues/22

### CODE REVIEW

**Comment by frewsxcv** [comment]
To use this:

`cargo install cargo-fuzz`

`cargo fuzz run <fuzz target name>`, in this case `cargo fuzz run parse`

if you want to add some files to use as seeds for the fuzzer, add them to `fuzz/corpus/parse/`. i recommend adding all the files from `test/` into there and then rerunning

File: fuzz/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (yurydelendik) [inline fuzz/.gitignore]: remove empty line

File: fuzz/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+[package]
+name = "wasmparser-fuzz"
+version = "0.0.1"
+authors = ["Automatically generated"]
```
Comment (yurydelendik) [inline fuzz/Cargo.toml]: nit: fix author name

**Review by yurydelendik — CHANGES_REQUESTED** [review]
Thank you.

I don't think we want to package fuzz into the crate package. Can you add exclude entry to the main Cargo.toml?

Can you squash commits? There is no reason to have "touch up" commits.

Please create a local README.md with the same instructions you provided in the PR comments. I was not able to run by following those instructions:

```
fuzz yury$ cargo fuzz run parse
error: could not decode the manifest file at "wasmparser.rs/wasmparser/Cargo.toml"
  caused by: redefinition of table `package` for key `package` at line 23
```

**Comment by frewsxcv** [comment]
> Can you squash commits? There is no reason to have "touch up" commits.

https://github.com/blog/2141-squash-your-commits

> Please create a local README.md with the same instructions you provided in the PR comments. I was not able to run by following those instructions:

Which directory did you run the command from? You should run it from the repository root, not from within the `fuzz/` directory.

**Comment by yurydelendik** [comment]
> https://github.com/blog/2141-squash-your-commits

Personally I'm trying not to do that: it removes signed commit verification and modifies origin of commit. I'm doing squashing only if contributors cannot to it himself. Would you like to squash it for you?

**Comment by yurydelendik** [comment]
> You should run it from the repository root, not from within the fuzz/ directory.

Different error when run from repo's dir: https://gist.github.com/yurydelendik/c59534507de07801fed90865854874e9

**Comment by frewsxcv** [comment]
Ah, you'll need to run nightly Rust for this. `rustup default nightly` if you're using rustup. I can add that to the README when I get around to it.

**Comment by yurydelendik** [comment]
>  I can add that to the README when I get around to it.

Yes, please. Can you add fuzzing section to the README?

**Review by yurydelendik — APPROVED** [review]

**Comment by frewsxcv** [comment]
Added a brief section to the README and squashed.

**Comment by yurydelendik** [comment]
Thank you for the patch

**Pull request merged**